### ISSUE TITLE
Error using yii2tech\filedb\Query;

### ISSUE THREAD

**Issue by gitrequests** [yii2tech/filedb#4]
If i use \yii2tech\filedb\ActiveRecord it is work fine.

But yii2tech\filedb\Query exit with error:
Call to a member function getQueryProcessor() on null
 in D:\Data\OpenServer\5.2.4\domains\site.my\vendor\yii2tech\filedb\Query.php at line 110 

Controller:

``` php
<?php

namespace app\controllers;

use Yii;
use yii\web\Controller;
use yii\filters\VerbFilter;

use yii2tech\filedb\Query;

class SiteController extends Controller
{

    public $layout = 'web';

    public function actionIndex()
    {
        //$items = Items::find()->all(); // <-- It is work fine!
        $query = new Query();
        $query->from('Items');
        $rows = $query->all(); // <-- Error!

        return $this->render('index', ['items' => $items]);
    }

    public function actionAbout()
    {
        return $this->render('about');
    }

}
```

filedb file (@app/data/Items.php):

``` php
<?php

return [
    1 => [
        'name' => 'admin',
        'description' => 'Site administrator',
    ],
    2 => [
        'name' => 'member',
        'description' => 'Registered front-end user',
    ],
];

```

Yii2 config:

``` php
//...
 'log' => [
            'traceLevel' => YII_DEBUG ? 3 : 0,
            'targets' => [
                [
                    'class' => 'yii\log\FileTarget',
                    'levels' => ['error', 'warning'],
                ],
            ],
        ],
        'db' => require(__DIR__ . '/db.php'),
        'filedb' => [
            'class' => 'yii2tech\filedb\Connection',
            'path' => '@app/data',
        ],
    ],
    'params' => $params,
//...
```

Did everything according to Your documentation.

**Comment by klimov-paul** [comment]
This happens because there is no default connection fetching inside the `Query`.
At the present state, you can bypass the issue passing `yii2tech\filedb\Connection` instance to the `Query::all()` method:

``` php
$query = new Query();
$query->from('Items');
$rows = $query->all(Yii::$app->filedb);
```

**Comment by klimov-paul** [comment]
Resolved by commit a183ab7c7411292f2a8b5d08f349c33815481faf

**Comment by gitrequests** [comment]
It work! Thx!
